packages/material-ui/src/Dialog/Dialog.js

```diff
@@ -240,6 +242,8 @@ const Dialog = React.forwardRef(function Dialog(props, ref) {
           <PaperComponent
             elevation={24}
             role="dialog"
+            aria-describedby={ariaDescribedby}
+            aria-labelledby={ariaLabelledby}
```

Comment:
What's the motivation for applying these aria props on the Paper?

Reply:
There's the role just one line above. They are not applied to the Paper specifically but the dialog. They were attached to a generic container before.

Reply:
Thanks for the precision